Runtime/Scripts/LyumaAv3Runtime.cs

```diff
@@ -2100,6 +2295,71 @@ void NormalUpdate()
 						defaultHeadScale = head.localScale;
 					}
 					head.localScale = EnableHeadScaling ? new Vector3(0.0001f, 0.0001f, 0.0001f) : defaultHeadScale; // head bone is set to 0.0001 locally (not multiplied
+
+					Type headChopType = Type.GetType("VRC.SDK3.Avatars.Components.VRCHeadChop, VRCSDK3A, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null");
+					if (headChopType != null)
+					{
+						Vector3 div(Vector3 x, Vector3 y) => new Vector3(x.x / y.x, x.y / y.y, x.z / y.z);
+						Vector3 mul(Vector3 x, Vector3 y) => new Vector3(x.x * y.x, x.y * y.y, x.z * y.z);
+						if (headChopData == null)
+						{
+							headChops = GetComponentsInChildren(headChopType);
```

Comment:
```suggestion
							headChops = GetComponentsInChildren(headChopType, true);
```

We should collect all components first and check enablement/activeness on each frame